blamo/lib/Export/email.dart

```diff
@@ -10,20 +11,41 @@ import 'package:permission_handler/permission_handler.dart';
 // Will throw exception if pdf/csv filepath is not found
 // TODO need to create backend to vacuum up all doc files and create pdf/csv based only on doc name
 Future<String> sendEmail(String documentName,String docType) async {
+  List<String> attachments = [];
+
   ObjectHandler objectHandler = new ObjectHandler();
   //String outputPath = "storage/emulated/0/Android/data/edu.oregonstate.blamo/files/output_test.pdf";
   String outputPath;
   debugPrint("EMAIL.dart - Send email");
 
-  try {
-    outputPath = await objectHandler.getPathToFile(documentName, docType);
-    debugPrint("output path: " + outputPath);
-  } catch (e) {
-    print("EMAIL.dart - No filepath found for document to send");
-    return "No $docType type file found for $documentName";
+  Future<bool> addAttachment(String doc, String docT) async {
+    try {
+      outputPath = await objectHandler.getPathToFile(doc, docT);
+      debugPrint("output path: " + outputPath);
+      attachments.add(outputPath);
+      return true;
+    } catch (e) {
+      print("EMAIL.dart - No filepath found for document to send");
+      return false;
+    }
+  }
+
+  if(docType == 'both'){
+    bool csv = await addAttachment(documentName,'csv');
+    bool pdf = await addAttachment(documentName, 'pdf');
+    if(csv == false){
+      return "$documentName.csv not found";
+    } else if(pdf == false) {
```

Comment:
Might consider removing the else, and building a concatenated string. The concatenated string could then be used to indicate to the user what part is not found, or both being empty. Something like:

if(csv == false) ---> S+= ".CSV"
if(pdf == false) ---> S+= ".PDF"

And when you recieve that in index.dart respond with:

if(returned == ".PDF") ---> indicate pdf
else if(returned == ".CSV") ---> indicate CSV
else if (returned == ".CSV.PDF") ---> indicate neither